### PR TITLE
chore: remove mnist use case

### PR DESCRIPTION
**Pull request by jfrery**
Closes https://github.com/zama-ai/concrete-ml-internal/issues/3353

### CODE REVIEW

**Review by fd0r — APPROVED** [review]
Not sure why but fine with me

File: README.md [hunk 1]
```diff
@@ -126,8 +126,6 @@ Various tutorials are proposed for the [built-in models](docs/built-in-models/ml
 - [Health diagnosis](use_case_examples/disease_prediction/): based on a patient's symptoms, history and other health factors, gives
   a diagnosis using FHE to preserve the privacy of the patient.
 
-- [MNIST](use_case_examples/mnist): a Python script and notebook showing quantization-aware training following FHE constraints. A fully-connected neural network is implemented with [Brevitas](https://github.com/Xilinx/brevitas) and is converted to FHE with Concrete ML.
```
Comment (bcm-at-zama) [inline README.md:129]: nothing to replace with? eg, in our presentations, we clearly say MNIST 97%, it would be nice that people can repro easily

Comment (jfrery) [reply README.md:129]: We have it in our example. I will push a 98% version of mnist that runs in 20 seconds. This is going to be 20 seconds better than the current CNN mnist but also 6% accuracy better.

Comment (bcm-at-zama) [reply README.md:129]: great, so can we keep a MNIST entry in the README, and point it there?

**Review by fd0r — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jfrery) [reply README.md:129]: Those only target the use case folder but yes maybe we can just redirect to the advanced example for mnist. WDYT @andrei-stoian-zama ?

**Review by RomanBredehoft — APPROVED** [review]
some remaining problems with docs but otherwise fine for me !

**Comment by jfrery** [comment]
along with the new CNN version for mnist -> https://github.com/zama-ai/concrete-ml/pull/276

**Review by fd0r — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (jfrery) [reply README.md:129]: What should we do about this @andrei-stoian-zama ? Should I add it back in https://github.com/zama-ai/concrete-ml/pull/276? or we remove mnist completely from the doc as well.

**Review by RomanBredehoft — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (andrei-stoian-zama) [reply README.md:129]: I think we remove this one. For custom networks there is CIFAR

Comment (jfrery) [reply README.md:129]: Alright let's merge

**Review by andrei-stoian-zama — APPROVED** [review]

**Review by kcelia — APPROVED** [review]
Thanks

**Review by fd0r — APPROVED** [review]

**Review by RomanBredehoft — APPROVED** [review]

**Pull request merged**